tests/model_hub_tests/performance_tests/conftest.py

```diff
@@ -9,19 +9,22 @@
 from models_hub_common.utils import round_num
 import json
 import requests
+from datetime import datetime
 
 
 def pytest_sessionstart(session):
     print('pytest_sessionstart')
     session.results = []
+    current_datetime = datetime.now()
+    session.start_time = current_datetime.strftime('%Y-%m-%d %H:%M')
 
 
 def pytest_sessionfinish(session, exitstatus):
     print('pytest_sessionfinish')
     dump_data = []
     for item in session.results:
         dump_item = dict()
-        dump_item['version'] = '1'
+        dump_item['version'] = session.start_time
```

Comment:
why `version`? The field name looks expect some version. May be it makes sense to rename field name?

Reply:
version is a required field name. it could be asked by team who develops OVVP